www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.php

```diff
@@ -147,8 +146,6 @@ public function performValidation($validateFullModel = false)
         $messages = parent::performValidation($validateFullModel);
         // 1. Check domain-port combinations
         $this->checkForUniquePortCombos($this->reverseproxy->reverse->iterateItems(), $messages);
-        // 2. Check subdomain-port combinations
-        $this->checkForUniquePortCombos($this->reverseproxy->subdomain->iterateItems(), $messages);
         // 3. Check that subdomains are under a wildcard or exact domain
```

Comment:
1., 3., 4.) ? ;)

Reply:
I didnt change the numbering in this PR since another PR would have resulted in merge conflicts. I will adjust it now.

Reply:
@fichtner Now there is this merge conflict.

Reply:
just go to github click "resolve conflicts"... it's confused about a new function it seems

Reply:
I solved it.

Reply:
👍